.buildkite/release_checks/scenarios/komodor_agent/scenario.py

```diff
@@ -0,0 +1,56 @@
+import os
+
+from scenario import Scenario
+import asyncio
+
+API = os.getenv("AGENT_API_KEY", '36d9e750-f50c-442d-ba32-1cec72a04d4c')
```

Comment:
@Moran-k :( Need to either cleanup the history or revoke this api key

Reply:
removed and revoked

Reply:
was A key from 'nir' test account